sdk/mixedreality/Azure.MixedReality.RemoteRendering/tests/Samples/RemoteRenderingConvertAssetSample.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.MixedReality.RemoteRendering.Tests.Samples
+{
+    // These tests assume that the storage account is accessible from the remote rendering account.
+    // See https://docs.microsoft.com/azure/remote-rendering/how-tos/create-an-account
+    // Since the roles can take a while to propagate, we do not live test these samples.
+
+    public class RemoteRenderingConvertAssetSample : SamplesBase<RemoteRenderingTestEnvironment>
+    {
+        private readonly RemoteRenderingAccount _account;
+        private readonly string _accountKey;
+        private readonly Uri _serviceEndpoint;
+
+        public RemoteRenderingConvertAssetSample()
+        {
+            _account = new RemoteRenderingAccount(TestEnvironment.AccountId, TestEnvironment.AccountDomain);
+            _accountKey = TestEnvironment.AccountKey;
+            _serviceEndpoint = new Uri(TestEnvironment.ServiceEndpoint);
+        }
+
+        //[Test]
```

Comment:
See `Explicit` attribute (https://docs.nunit.org/articles/nunit/writing-tests/attributes/explicit.html).


Example: https://github.com/Azure/azure-sdk-for-net/blob/69f25da98b5210f1b850fb079f48916787aca3e9/sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs#L835